src/main/Plugins/ValidationPlugins/Dns/ILookupClientProvider.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Net;
```

Comment:
I'd rather put this interface in Services/Interfaces and the implementation in Services.

Reply:
fixed in https://github.com/PKISharp/win-acme/pull/1054/commits/110d1587dab198dc3d173189078b794fc1b26551

Reply:
Thanks!